FiveGuyes/FiveGuyes/Sources/Views/Components/WeeklyReadingProgressView.swift

```diff
@@ -12,22 +12,32 @@ struct WeeklyReadingProgressView: View {
     @Query(filter: #Predicate<UserBook> { $0.isCompleted == false })
     private var currentlyReadingBooks: [UserBook]  // 현재 읽고 있는 책을 가져오는 쿼리
     
+    // 텍스트 변경을 위한 추가 필요데이터
+    let today = Date()
+    
     var body: some View {
+        // 텍스트 변경 기능을 위한 추가 코드
+        // 현재 읽는 책을 가져옵니다
         if let currentReadingBook = currentlyReadingBooks.first {
+            // getTodayRedcrodedPage함수를 통해 오늘 기록을 가져옵니다
+            let todayRecords = getTodayRecordedPage(for: currentReadingBook, from: today)
             VStack(alignment: .leading, spacing: 17) {
-                
                 VStack(alignment: .leading, spacing: 8) {
-                    Text("\(currentReadingBook.nonZeroReadingDaysCount())일 째 도전중!")
-                        .font(.system(size: 17, weight: .semibold))
-                        .foregroundColor(.black)
-                    
+                    // 텍스트 변경을 위한 추가 코드
+                    // todayRecords이 nil이 아니면 todayRecords가 할당
+                        if let todayRecords = todayRecords {
```

Comment:
```suggestion
                        if let todayRecords {
```
옵셔널 벗길 때 변수명 같게 쓰는 경우는 그대로 써도 됩니다!

Reply:
어색해보였는데 알려주셔서 너무 감사합니다! +_+ 요것도 반영하고 바로 푸쉬해서 머지할게요!